recipes/xorg-libxxf86vm/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set xorg_name = "libXxf86vm" %}
+{% set xorg_category = "lib" %}
+{% set name = "xorg-" ~ xorg_name %}
+{% set version = "1.1.5" %}
+{% set sha256 = "247fef48b3e0e7e67129e41f1e789e8d006ba47dba1c0cdce684b9b703f888e7" %}
+{% set am_version = "1.15" %} # keep synchronized with build.sh
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://www.x.org/releases/individual/{{ xorg_category }}/{{ xorg_name }}-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - m2-autoconf  # [win]
+    - m2-automake{{ am_version }}  # [win]
+    - m2-libtool  # [win]
+    - m2w64-pkg-config  # [win]
+    - make
+    - pkg-config  # [not win]
+    - posix  # [win]
+    - gnuconfig  # [unix]
+    - {{ compiler('c') }}        # [unix]
+    - {{ compiler('m2w64_c') }}  # [win]
+    - make                          # [unix]
+    - autoconf                      # [unix]
+    - automake                      # [unix]
+    - gettext                       # [unix]
+    - libtool                       # [unix]
+  host:
+    - xorg-util-macros
+    - xorg-xproto
+    - xorg-libx11
+    # Additional constraintless xorg-libx11 dep below is for conda-build to add potential pinning to.
```

Comment:
The comment says "below" but the constraintless dep is "above", here.

Reply:
thanks.